test/fixture-read-config/with_dependency/terragrunt.hcl

```diff
@@ -0,0 +1,9 @@
+locals {
+  config_with_dependency = read_terragrunt_config("${get_terragrunt_dir()}/dep/terragrunt.hcl")
+}
+
+terraform {
+  source = "${get_terragrunt_dir()}/../../fixture-inputs"
+}
+
+inputs = local.config_with_dependency.dependency.inputs.outputs
```

Comment:
NIT: Should we have a test case that checks that we're able to access every aspect of the config read in via `read_terragrunt_config`?

E.g., 

```hcl
locals {
  config = read_terragrunt_config("foo.hcl")
}

inputs = {
  locals = local.config.locals
  terraform = local.config.terraform
  inputs = local.config.inputs
  # etc
}
```

And the test checks that all the values in `inputs` are set to the exact values in `foo.hcl`?

Reply:
Done: bbf042e